.github/workflows/test-integration.yml

```diff
@@ -1,22 +1,22 @@
 name: test:integration
+
 on:
   push:
-    branches: [main, master]
+    branches: main
```

Comment:
Why main and not master? Not trying to start an argument about being PC, but I'm a bit confused why we seemingly used to be able to use both interchangeably but now need to limit ourselves to one,